### PR TITLE
Wizard

### PR DESCRIPTION
**Pull request by kkadosa**
Only look at the WizardTest project.

### CODE REVIEW

File: WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/EObject.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import javax.xml.bind.annotation.XmlType;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "EObject")
+public class EObject {
+
+    public void subCompartmentalize(Path parent, Element topParent, Path topPath) throws IOException {
```
Comment (gkulcsar) [inline WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/EObject.java]: missing override?

File: WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/EStringToStringMapEntry.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "EStringToStringMapEntry")
+public class EStringToStringMapEntry extends EObject {
+
+    @XmlAttribute(name = "key")
+    protected String key;
+    @XmlAttribute(name = "value")
+    protected String value;
+
+    private void subCompartmentalize(EStringToStringMapEntry ss, Path parent, Element topParent, Path topPath) throws IOException {
```
Comment (gkulcsar) [inline WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/EStringToStringMapEntry.java]: missing override?

File: WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/EStructuralFeature.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
+//
```
Comment (gkulcsar) [inline WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/EStructuralFeature.java]: why don't you have... ?

File: WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/ETypedElement.java [hunk 4]
```diff
@@ -0,0 +1,114 @@
+//
```
Comment (gkulcsar) [inline WizardTest/src/main/java/org/eclipse/emf/ecore/jaxbmodel/ETypedElement.java]: why... ? :)

**Comment by kkadosa** [comment]
I think I'll go for a generalized solution

**Pull request closed without merging**